package.json

```diff
@@ -4,7 +4,6 @@
     "packages/*"
   ],
   "scripts": {
-    "postinstall": "yarn radix:build",
```

Comment:
We must have this in place, otherwise when `yarn start` is run for the first time (on a fresh clone) the website will be broken, since there is no `dist` folder in `packages/radix`

This will also break CI on a clean build (at the moment it works because its cached - I believe)

Reply:
Yeah I saw that - monorepos don't usually have a `start` command as there are often too many concerns across their contents. You may want to move it down to the `website` workspace (`prestart`).